trackman/templates/library/labels.html

```diff
@@ -19,6 +19,14 @@ <h1>Library</h1>
     <li class="nav-item"><a class="nav-link" href="{{ url_for('trackman_library.fixup') }}">Fixup</a></li>
 </ul>
 
+<!-- TODO beautify -->
+<h2>search<h2>
+    <p>Only searches exact matches</p>
+    <form action=label>
```

Comment:
Please quote this and use the url_for Jinja2 filter instead. Also, are you sure this works? This should send a POST request and that should be rejected without a CSRF token.